stock_account_valuation_report/models/product_product.py

```diff
@@ -30,29 +30,11 @@ class ProductProduct(models.Model):
         compute="_compute_inventory_value",
         search="_search_qty_discrepancy",
     )
-    valuation = fields.Selection(
```

Comment:
Reviewed and understood reason for deletion. Field is now related on product.template to categ_id and directly accessible to product.